packages/calcite-components/src/components/tooltip/TooltipManager.ts

```diff
@@ -177,7 +190,11 @@ export default class TooltipManager {
     this.toggleTooltip(tooltip, true);
   };
 
-  private blurHandler = (): void => {
+  private blurHandler = (event: FocusEvent): void => {
+    if (event.defaultPrevented) {
```

Comment:
This isn't necessary as the blur event isn't cancelable (see [spec](https://www.w3.org/TR/uievents/#event-type-blur) and [demo](https://codepen.io/jcfranco/pen/myeVdYp)).